plugins/qiskit_executor/schemas.py

```diff
@@ -12,25 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Optional
 import marshmallow as ma
+from celery.utils.log import get_task_logger
 from qhana_plugin_runner.api.util import (
     FrontendFormBaseSchema,
     FileUrl,
 )
-from celery.utils.log import get_task_logger
+from qhana_plugin_runner.util.logging import redact_log_data
 
 
 TASK_LOGGER = get_task_logger(__name__)
 
 
-class CircuitSelectionInputParameters:
+class CircuitParameters:
```

Comment:
This is a prime use case of dataclasses. (though keep in mind that dataclasses will have `__str__` and `__repr__` methods by default!)
Also true for other simple classes in this file